frontends/web/src/routes/device/setup/seed-create-new.jsx

```diff
@@ -153,7 +153,10 @@ export default class SeedCreateNew extends Component {
                         onClick={goBack}>
                         {t('button.back')}
                     </Button>
-                    <Button primary onClick={this.handleStart}>
+                    <Button
+                        primary
+                        onClick={this.handleStart}
```

Comment:
If you can click the button before it is disabed, `checkSDcard()` is called and starts a new timer-loop, even though one is already running.

Reply:
fixed in 44c5acfa2d66bf64f9a605bde5c65c45e85c6436